cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1441,7 +1445,9 @@ def make_resource(self, data, **kwargs):
 class SchedulerPluginUserSchema(BaseSchema):
     """Represent the schema of the Scheduler Plugin."""
 
-    name = fields.Str(required=True, metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
+    name = fields.Str(
```

Comment:
I think it's better to have a config validator here and not only avoid the new lines, but validate the name as unix user name. You can use the same regex used here https://github.com/aws/aws-parallelcluster-cookbook/blob/4756da1f9660bcfb94cd8f59a7e5b3e8886e67b4/cookbooks/aws-parallelcluster-config/files/default/dcv/pcluster_dcv_authenticator.py#L130

Reply:
Added